From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 3 Sep 2010 16:41:09 -0400
Subject: [net] bnx2x: use DMA_BIT_MASK(64) over DMA_64BIT_MASK
Message-id: <20100903164109.25415.33234.stgit@brian.englab.brq.redhat.com>
Patchwork-id: 28076
O-Subject: [RHEL5.6 BZ572012 PATCH 01/46] bnx2x: replace all DMA_64BIT_MASK
	macro with DMA_BIT_MASK(64)
Bugzilla: 572012
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>

from upstream 6a35528a8346f6e6fd32ed7e51f04d1fa4ca2c01

diff --git a/drivers/net/bnx2x_main.c b/drivers/net/bnx2x_main.c
index b8b5266..3bf2771 100644
--- a/drivers/net/bnx2x_main.c
+++ b/drivers/net/bnx2x_main.c
@@ -11870,9 +11870,9 @@ static int __devinit bnx2x_init_dev(struct pci_dev *pdev,
 		goto err_out_release;
 	}
 
-	if (pci_set_dma_mask(pdev, DMA_64BIT_MASK) == 0) {
+	if (pci_set_dma_mask(pdev, DMA_BIT_MASK(64)) == 0) {
 		bp->flags |= USING_DAC_FLAG;
-		if (pci_set_consistent_dma_mask(pdev, DMA_64BIT_MASK) != 0) {
+		if (pci_set_consistent_dma_mask(pdev, DMA_BIT_MASK(64)) != 0) {
 			printk(KERN_ERR PFX "pci_set_consistent_dma_mask"
 			       " failed, aborting\n");
 			rc = -EIO;
